common/src/test/java/com/netflix/conductor/common/workflow/WorkflowDefTest.java

```diff
@@ -304,12 +304,7 @@ public void testWorkflowTaskInputParamValueInvalid() {
 		ValidatorFactory factory = Validation.buildDefaultValidatorFactory();
 		Validator validator = factory.getValidator();
 		Set<ConstraintViolation<Object>> result = validator.validate(workflowDef);
-		assertEquals(1, result.size());
-
-		List<String> validationErrors = new ArrayList<>();
-		result.forEach(e -> validationErrors.add(e.getMessage()));
-
-		assertTrue(validationErrors.contains("key: blabla input parameter value: is null or empty"));
+		assertEquals(0, result.size());
 	}
 
```

Comment:
Consider adding a test case for `listInputParamWithEmptyString` to this class as well.

Reply:
done